lib/ColorThief/Image/Adapter/GmagickImageAdapter.php

```diff
@@ -15,6 +15,12 @@ public function load($resource)
             throw new \InvalidArgumentException("Passed variable is not an instance of Gmagick");
         }
 
+        if ($resource->getImageColorSpace() == Gmagick::COLORSPACE_CMYK) {
```

Comment:
I'm not sure we should limit the conversion to CMYK images. At first glance, I found the first implementation fine, as it also covers other color spaces that are probably affected by the same problem. That said, this would require further testing and for now it may be more reasonable to remain limited to the CMYK colorspace as done here.

Reply:
Agreed. Conversion to RGB may be relevant in other cases, but I dare not assume which.